unstable/lens/struct-list.scrbl

```diff
@@ -0,0 +1,23 @@
+#lang scribble/manual
+
+@(require lens/private/doc-util/main)
+
+@title{Converting between structs and lists}
+
+@deftogether[[
+  @defform[(struct->list-lens struct-id)]
+  @defform[(list->struct-lens struct-id)]]]{
+Lenses that convert between structs and lists.
+
+@lens-unstable-examples[
+  (struct foo (a b c) #:transparent)
+  (lens-view (struct->list-lens foo) (foo 1 2 3))
+  (lens-set (struct->list-lens foo) (foo 1 2 3) '(4 5 6))
+  (lens-view (list->struct-lens foo) '(1 2 3))
+  (lens-set (list->struct-lens foo) '(1 2 3) (foo 4 5 6))
+  (struct bar foo (d e))
```

Comment:
Could we split these into two example blocks, and before the block with inheritance describe whether the inherited fields come before or after the substruct fields in the list?